Inputs/SystematicsTest.yaml

```diff
@@ -77,7 +77,7 @@ Systematics:
     SplineInformation:
       Mode: [0]
       InterpolationType: TSpline3
-      SplineName: Spline_Param_0
+      SplineName: Spline_0
     DetID: 985
```

Comment:
Why was this chaged?

Reply:
These are the names in the spline root file
<img width="460" alt="testSplineFile" src="https://github.com/user-attachments/assets/4ce7c701-8a7a-49c7-b1eb-621e48c55c53">

I think it just didn't cause an issue before as the c++ samplePDF in the tutorial doesn't use these splines